src/Pokemon.jsx

```diff
@@ -1,22 +1,24 @@
 import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
 import { Typography, Link, CircularProgress, Button } from '@material-ui/core';
-import { toFirstCharUppercase } from './constants';
+import { toFirstCharUppercase } from './utils/constants';
 import axios from 'axios';
 
 const Pokemon = (props) => {
-  const { match, history } = props;
+  const { match } = props;
   const { params } = match;
   const { pokemonId } = params;
-  const [pokemon, setPokemon] = useState(undefined);
+  const history = useHistory();
+  const [pokemon, setPokemon] = useState();
```

Comment:
Wasn't sure why this was undefined. Generally, you never want to check that. It would be `null` if anything.